### PR TITLE
[now-dev] Add checking of RegExp src with beginning slash

### PR DESCRIPTION
**Pull request by ijjk**
This adds fallback checking including the beginning slash for RegExp's that rely on the beginning slash being there

### CODE REVIEW

File: src/commands/dev/lib/dev-router.ts [hunk 1]
```diff
@@ -71,7 +71,8 @@ export default async function(
 
       const keys: string[] = [];
       const matcher = PCRE(`%${src}%i`, keys);
-      const match = matcher.exec(reqPathname);
+      let match = matcher.exec(reqPathname);
```
Comment (styfle) [inline src/commands/dev/lib/dev-router.ts]: I think you can do this in one line:

```js
let match = matcher.exec(reqPathname) || matcher.exec(`/${reqPathname}`);
```

Or perhaps even make the optional `/` part of the regex.

Comment (ijjk) [reply src/commands/dev/lib/dev-router.ts]: Updated it to one line, I think it's good to check for this even if we did update the dynamic routing RegExp to have the beginning slash as optional.

**Review by styfle — APPROVED** [review]
Looks fine but let's have @TooTallNate review as well

**Review by TooTallNate — CHANGES_REQUESTED** [review]
This needs a unit test.

But also, I don't really see how this scenario can get hit considering that the leading slash is removed from the regexp before invoking the match. I'll wait for the unit test to understand more clearly.

**Comment by ijjk** [comment]
Just added a unit test for it

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**